hw1/02-palindrome.html

```diff
@@ -6,20 +6,26 @@
     <title>02 - Palindrome</title>
     <link
       rel="stylesheet"
-      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css" />
+      href="https://cdn.jsdelivr.net/npm/bootstrap@5.0.2/dist/css/bootstrap.min.css"
+    />
     <script defer src="02-palindrome.js"></script>
   </head>
-  <body>
+  <body style="background-color: blue">
     <main>
-      <section>
-        <h1>Exercise 02 - Palindrome</h1>
-        <div>
-          <label>Enter a positive number:</label>
-          <input type="number" />
+      <section
+        style="background-color: white"
+        class="position-absolute top-50 start-50 translate-middle p-3 w-50"
+      >
+        <h1 class="text-center">Exercise 02 - Palindrome</h1>
+        <div class="row pt-4">
+          <label class="col-5">Enter a positive number:</label>
+          <input class="col-6" type="number" />
```

Comment:
I believe this `label` element isn't properly associated to the `input` element.
I recommend explicitly referencing the associated input element with the `for` attribute: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label#attributes

Reply:
yes totally forgot about this! good catch.